src/main/scala/sbtghactions/PREventType.scala

```diff
@@ -16,23 +16,23 @@
 
 package sbtghactions
 
```

Comment:
Can we add documentation with the `@see` to the equivalent github documentation (incase something changes/breaks), see https://github.com/sbt/sbt-github-actions/blob/main/src/main/scala/sbtghactions/PermissionScope.scala#L21-L23 for an example